addons/mail/static/src/components/attachment_card/attachment_card.scss

```diff
@@ -90,7 +93,7 @@
     color: $o-brand-primary;
 }
 
-.o_AttachmentCard_extension {
+.o_AttachmentCard_extension, .o_AttachmentCard_separator, .o_AttachmentCard_formattedSize {
```

Comment:
If you want to share style among many elements, it's a sign all of them represent something meaningful.

Could introduce another structural classname, e.g. `o_AttachmentCard_detailsItem` and put style only on this structural classname. All elements that should behave like a `detailsItem` should have this classname, in addition to any other structural classname they have (ideally 1 of them should uniquely identity the part).

(FYI, this shared structural classname is kind of a "trait" for styling purpose. In another PR I said the model framework is not ready to properly introduce behaviour sharing or records... the end goal is for behaviour sharing to include complex record behavior, but also styling of parts in template! So in a sense, this is premise work to help introducing traits :) )